api/bazel/repositories.bzl

```diff
@@ -1,311 +1,22 @@
-    srcs = [
-        "google/api/annotations.proto",
-        "google/api/http.proto",
-    ],
-    visibility = ["//visibility:public"],
-)
-
-go_proto_library(
-    name = "descriptor_go_proto",
-    importpath = "github.com/golang/protobuf/protoc-gen-go/descriptor",
-    proto = "@com_google_protobuf//:descriptor_proto",
-    visibility = ["//visibility:public"],
-)
-
-proto_library(
-    name = "http_api_protos_proto",
-    srcs = [":http_api_protos_src"],
-    deps = ["@com_google_protobuf//:descriptor_proto"],
-    visibility = ["//visibility:public"],
-)
-
-cc_proto_library(
-    name = "http_api_protos",
-    srcs = [
-        "google/api/annotations.proto",
-        "google/api/http.proto",
-    ],
-    default_runtime = "@com_google_protobuf//:protobuf",
-    protoc = "@com_google_protobuf//:protoc",
-    deps = ["@com_google_protobuf//:cc_wkt_protos"],
-    visibility = ["//visibility:public"],
-)
-
-py_proto_library(
-    name = "http_api_protos_py",
-    srcs = [
-        "google/api/annotations.proto",
-        "google/api/http.proto",
-    ],
-    include = ".",
-    default_runtime = "@com_google_protobuf//:protobuf_python",
-    protoc = "@com_google_protobuf//:protoc",
-    visibility = ["//visibility:public"],
-    deps = ["@com_google_protobuf//:protobuf_python"],
-)
-
-go_proto_library(
-    name = "http_api_go_proto",
-    importpath = "google.golang.org/genproto/googleapis/api/annotations",
-    proto = ":http_api_protos_proto",
-    visibility = ["//visibility:public"],
-    deps = [
-      ":descriptor_go_proto",
-    ],
-)
-
-filegroup(
-     name = "rpc_status_protos_src",
-     srcs = [
-         "google/rpc/status.proto",
-     ],
-     visibility = ["//visibility:public"],
-)
-
-proto_library(
-     name = "rpc_status_protos_lib",
-     srcs = [":rpc_status_protos_src"],
-     deps = ["@com_google_protobuf//:any_proto"],
-     visibility = ["//visibility:public"],
-)
-
-cc_proto_library(
-     name = "rpc_status_protos",
-     srcs = ["google/rpc/status.proto"],
-     default_runtime = "@com_google_protobuf//:protobuf",
-     protoc = "@com_google_protobuf//:protoc",
-     deps = [
-         "@com_google_protobuf//:cc_wkt_protos"
-     ],
-     visibility = ["//visibility:public"],
-)
-
-go_proto_library(
-    name = "rpc_status_go_proto",
-    importpath = "google.golang.org/genproto/googleapis/rpc/status",
-    proto = ":rpc_status_protos_lib",
-    visibility = ["//visibility:public"],
-    deps = [
-      "@com_github_golang_protobuf//ptypes/any:go_default_library",
-    ],
-)
-
-py_proto_library(
-     name = "rpc_status_protos_py",
-     srcs = [
-         "google/rpc/status.proto",
-     ],
-     include = ".",
-     default_runtime = "@com_google_protobuf//:protobuf_python",
-     protoc = "@com_google_protobuf//:protoc",
-     visibility = ["//visibility:public"],
-     deps = ["@com_google_protobuf//:protobuf_python"],
-)
-""",
+        build_file = "//api/bazel:googleapis.BUILD",
```

Comment:
@lizan I thought `:googleapis.BUILD` would work so I'm fully qualifying the build_files. I'm unsure if this would work in the `data-plane-api`.

If not I'll put the content back inline. Was hoping to just clean up this file.

Reply:
I think you'll need `repository` parameter (see what's done for envoy_cc_library etc) and plug them accordingly